### ISSUE TITLE
Set the server to return logits

### ISSUE THREAD

**Issue by StellaAthena** [EleutherAI/GPTNeo#85]

**Comment by sdtblck** [comment]
this is an API thing, which i'll release in a different repo soon. So closing this for now.
